third_party/2and3/typing_extensions.pyi

```diff
@@ -19,6 +19,8 @@ _F = TypeVar('_F', bound=Callable[..., Any])
 _TC = TypeVar('_TC', bound=Type[object])
 class _SpecialForm:
     def __getitem__(self, typeargs: Any) -> Any: ...
+def runtime_checkable(cls: _TC) -> _TC: ...
+# This is just an alias for above kept for backwards compatibility.
```

Comment:
Why not write it as `runtime = runtime_checkable`?

Reply:
IIRC pytype and/or PyCharm didn't like function aliases in stubs, for example the `attrs` entry in https://github.com/python/typeshed/blob/master/tests/pytype_blacklist.txt may be related to this.

@JelleZijlstra If you think it should ne OK, I can update this to use an actual alias.

Reply:
I don't recall that being an issue; it may have been fixed. In any case as far as I know other type checkers don't tend to use typing.pyi from typeshed.

Reply:
It looks like it was only for overloads https://youtrack.jetbrains.com/issue/PY-27788. I will update now.